Jetsnack/app/src/main/java/com/example/jetsnack/ui/components/Filters.kt

```diff
@@ -79,31 +89,64 @@ fun FilterChip(
 ) {
     val (selected, setSelected) = filter.enabled
     val backgroundColor by animateColorAsState(
-        if (selected) JetsnackTheme.colors.brand else JetsnackTheme.colors.uiBackground
+        if (selected) JetsnackTheme.colors.brandSecondary else JetsnackTheme.colors.uiBackground
     )
     val border = Modifier.fadeInDiagonalGradientBorder(
         showBorder = !selected,
         colors = JetsnackTheme.colors.interactiveSecondary,
         shape = shape
     )
     val textColor by animateColorAsState(
-        if (selected) JetsnackTheme.colors.textInteractive else JetsnackTheme.colors.textSecondary
+        if (selected) Color.Black else JetsnackTheme.colors.textSecondary
```

Comment:
Yes, for both themes the selected chip has a Black text, I didn't used a Theme color because there is no color that remains black for both variants.

![image](https://user-images.githubusercontent.com/16503982/116610032-aea0db80-a8fa-11eb-8b28-12d6b1fcb0da.png)